README.md

```diff
@@ -7,17 +7,18 @@ Yet another F-Droid client.
 
 ## Description
 
-Unofficial F-Droid client that resembles the classic F-Droid client.
+Unofficial F-Droid client in the style of the classic one.
 
-The client supports all basic F-Droid features, such as repository management and package installation. It doesn't
-support privileged extension, root installation, or local repositories.
+Jump over the lazy dog, manage repositories and install software quickly.
```

Comment:
Shouldn't there be a comma before "and"?

Reply:
Nice catch!